src/main/webapp/js/instructorStudentList.es6

```diff
@@ -323,6 +324,12 @@ const seeMoreRequest = function (e) {
         // Do not make ajax call if students shown already above limit
         showStudentLimitError(courseCheck, displayIcon);
     }
+
+    if ($(panelCollapse).attr('class').indexOf('checked') === -1) {
+        $(toggleCheveron).addClass('glyphicon-chevron-down').removeClass(' glyphicon-chevron-up');
+    } else {
+        $(toggleCheveron).addClass('glyphicon-chevron-up').removeClass(' glyphicon-chevron-down');
```

Comment:
@wkurniawan07 I tried to use the `toggleChevron` function defined inside [common.es6](https://github.com/TEAMMATES/teammates/blob/master/src/main/webapp/js/common.es6#L988), but can't use that because whenever the user click on the `panel-heading` of div, the `seeMoreRequest` function is called which again triggers the `panel-heading` of div due to which the `toggleChevron` function is called twice in just one click of user on `panel-heading`. 

That simply means the `toggleChevron` is called twice even when the user clicks on the panel just one time.